SwitchSpeak-iOS/SwitchSpeak-iOS/TouchGrid/ButtonNode.swift

```diff
@@ -41,36 +32,54 @@ class ButtonNode: Node {
         self.cardData = cardData
         
         self.button.setTitle(cardData.text, for: .normal)
+        self.button.titleLabel!.numberOfLines = 0; // Dynamic number of lines
+        self.button.titleLabel!.lineBreakMode = .byWordWrapping;
+        
+        let fontSize:CGFloat = GlobalSettings.getUserSettings().getFontSize()
+        let sizedFont:UIFont = self.button.titleLabel!.font!.withSize(fontSize)
+        self.button.titleLabel!.font = sizedFont
+        
+        if let cardImage:UIImage = UIImage(data:cardData.imagefile) {
+            let buttonWidth:CGFloat = self.button.frame.width
+            let buttonHeight:CGFloat = self.button.frame.height
+            let buttonAR:CGFloat = buttonWidth / buttonHeight // Button Aspect Ratio
+            let _buttonAR:CGFloat = 1.0 / buttonAR
+            
+            let imageBorder:CGFloat = fontSize
```

Comment:
Was just wondering why you choose `fontSize` as the image border width?

Reply:
It was just convenient. I wonder if putting `fontSize * 1.0` would be clearer here...